src/axom/quest/DistributedClosestPoint.hpp

```diff
@@ -559,6 +559,11 @@ class DistributedClosestPointImpl
       PointType lower(&recvbuf[i * 2 * DIM]);
       PointType upper(&recvbuf[i * 2 * DIM + DIM]);
       all_aabbs.emplace_back(BoxType(lower, upper));
+      // Correct the auto-correction in the BoundingBox constructor.
```

Comment:
This is needed because the `BoundingBox` constructor switches the `lower` and `upper` points to make the box valid.  We are actually interested in communicating that the box is invalid--it indicates that the domain has no bounding box (domain underloading).

Reply:
Suggestion:

Could you please modify the `BoundingBox` constructor to take a third optional parameter which controls whether the bounds get fixed? The default should be `true` to preserve the current behavior. 

E.g. something like this:
```cpp
AXOM_HOST_DEVICE
  BoundingBox(const PointType& lowerPt, 
              const PointType& upperPt,
              bool shouldFixBounds = true)
    : m_min(lowerPt)
    , m_max(upperPt)
  {
    if(shouldFixBounds) { this->checkAndFixBounds(); }
  }
```

Reply:
Done.